mod-record-specifications-validator/src/main/java/org/folio/rspec/validation/validator/marc/impl/MarcRecordRuleValidator.java

```diff
@@ -15,22 +22,33 @@
 import org.folio.rspec.validation.validator.SpecificationRuleValidator;
 import org.folio.rspec.validation.validator.marc.model.MarcField;
 import org.folio.rspec.validation.validator.marc.model.MarcRecord;
+import org.folio.rspec.validation.validator.marc.model.MarcRuleCode;
 
 public class MarcRecordRuleValidator implements SpecificationRuleValidator<MarcRecord, SpecificationDto> {
 
+  private static final List<MarcRuleCode> FIELD_RULES = List.of(
+    NON_REPEATABLE_FIELD
+  );
+  private static final List<MarcRuleCode> FIELD_SET_RULES = List.of(
+    UNDEFINED_FIELD,
+    MISSING_FIELD,
+    INVALID_FIELD_TAG,
+    NON_REPEATABLE_1XX_FIELD,
+    NON_REPEATABLE_REQUIRED_1XX_FIELD
+  );
```

Comment:
Looks like overhead. Instead of defining two lists of validators you have two lists of rules, two separate methods for providing rules and switches in each of them.

Reply:
Probably, I've tried to split the code a little. I will revert it.

Reply:
Maybe will change something when we have all validators. Reverted for now